lib_test/jbuild

```diff
@@ -25,7 +25,7 @@ let coverage_rewriter ~full =
 let () = Printf.ksprintf Jbuild_plugin.V1.send {|
 (executable
  ((name suite)
-  (libraries (lwt.unix nbd oUnit))
+  (libraries (lwt.unix nbd oUnit io-page.unix))
```

Comment:
For some reason I had to add this to avoid a linking error :/

Reply:
Please sort the library names in `libraries`

Reply:
Thanks, good point. Done.